File: EDA Project/fix_php_session.py
import os

files = ['login.php', 'signup.php', 'create-account.php']

for file in files:
    with open(file, 'r', encoding='utf-8') as f:
        lines = f.readlines()
        
    start_idx = -1
    end_idx = -1
    for i, line in enumerate(lines):
        if '<?php' in line and start_idx == -1:
            start_idx = i
        if '?>' in line and start_idx != -1 and end_idx == -1:
            end_idx = i
            break
            
    if start_idx != -1 and end_idx != -1:
        php_lines = lines[start_idx:end_idx+1]
        del lines[start_idx:end_idx+1]
        new_lines = php_lines + ['\n'] + lines
        
        with open(file, 'w', encoding='utf-8') as f:
            f.writelines(new_lines)
        print(f"Fixed {file}")

File: EDA Project/fix_php_session.php
<?php
$files = ['login.php', 'signup.php', 'create-account.php'];

foreach ($files as $file) {
    $lines = file($file);
    $start_idx = -1;
    $end_idx = -1;
    
    foreach ($lines as $i => $line) {
        if (strpos($line, '<?php') !== false && $start_idx == -1) {
            $start_idx = $i;
        }
        if (strpos($line, '?>') !== false && $start_idx != -1 && $end_idx == -1) {
            $end_idx = $i;
            break;
        }
    }
    
    if ($start_idx != -1 && $end_idx != -1) {
        $php_lines = array_slice($lines, $start_idx, $end_idx - $start_idx + 1);
        array_splice($lines, $start_idx, $end_idx - $start_idx + 1);
        
        $new_lines = array_merge($php_lines, ["\n"], $lines);
        file_put_contents($file, implode("", $new_lines));
        echo "Fixed $file\n";
    }
}
